source/features/next-scheduled-github-action.tsx

```diff
@@ -65,14 +65,19 @@ async function init(): Promise<false | void> {
 
 		const nextTime = parseCron.nextDate(workflows[workflowName]);
 		if (nextTime) {
-			workflowListItem.append(<em>(next <relative-time datetime={nextTime.toString()}/>)</em>);
+			const relativeTime = <relative-time datetime={nextTime.toString()}/>;
+			workflowListItem.append(<em>(next {relativeTime})</em>);
+			workflowListItem.parentElement!.classList.add('tooltipped', 'tooltipped-n');
+			workflowListItem.parentElement!.setAttribute('aria-label', 'Next ' + relativeTime.textContent!.toString());
 		}
 	}
 }
 
 void features.add(__filebasename, {
 	include: [
-		pageDetect.isRepositoryActions
+		pageDetect.isRepositoryActions,
+		// Enable feature on the pages of selected workflows
+		() => /actions\/workflows\/.+\.ya?ml$/.test(location.pathname)
```

Comment:
I think this actually should be included in `isRepositoryActions` like it was before. GitHub just changed the URL. It used to be `/actions?query=` and now it's `/actions/workflows`, so it no longer matches this page.

Reply:
This PR can be merged after this line is dropped from here and `github-url-detection` is updated here as well.